VideoLocker/src/main/java/org/edx/mobile/util/DeviceProfileUtil.java

```diff
@@ -20,4 +20,18 @@ public static String getModelName(){
     public static String getOSType(){
         return Build.VERSION.RELEASE;
     }
+
+    public static String getDeviceProfileInfo(){
+        String manufacturerName = getManufacturerName();
+        String modelName = getModelName();
+        String osType = getOSType();
+        return manufacturerName +  ":" + modelName + ":" + osType;
+    }
+
+    public static boolean isSamsungGalaxyS3(){
```

Comment:
@aleffert there is a bug related to SamsungGalaxyS3, we may need to handle it. Just added as a help method